server/services/config/conditions/v2/conditions.ts

```diff
@@ -47,13 +65,21 @@ export const conditions: ConditionMetadata[] = [
   {
     id: 'ATTEND_ALL',
     text: 'Attend all appointments arranged for you with [INSERT NAME], a [PSYCHIATRIST / PSYCHOLOGIST / MEDICAL PRACTITIONER] and co-operate fully with any care or treatment they recommend.',
-    user_input: 'appointmentName', // need to change?!
+    user_input: 'attendAll',
     field_position: {
       appointmentName: 0,
-      appointmentProfession: 1,
+      appointmentProfessions: 1,
     },
     group_name: GroupName.MAKING_OR_MAINTAINING_CONTACT,
     subgroup_name: null,
+    displayForEdit: (inputtedCondition) => {
+      const result = inputtedCondition.appointmentProfessions
+      const appointmentProfessions = typeof result === 'string' ? [result] : result
+      return {
+        ...inputtedCondition,
+        appointmentProfessions,
+      }
```

Comment:
need to handle what happens when a single value is passed back - its relatively easy to handle this in the html but didn't want to be inconsistent with the other forms